k2vr-installer-gui/Tools/Utils.cs

```diff
@@ -20,40 +20,58 @@ public static bool IsDirectoryEmpty(string path)
         public static bool EnsureSteamVrClosed()
         {
             Logger.Log("Checking if SteamVR is open...", false);
-            foreach (Process process in Process.GetProcesses())
+
+            //Check if SteamVR is running
+            if (Process.GetProcesses().FirstOrDefault((Process proc) => proc.ProcessName == "vrserver" || proc.ProcessName == "vrmonitor") == null)
+            {
+                return true;
+            }
+
+            //Ask if SteamVR should be closed
+            if (MessageBox.Show(
+                "SteamVR needs to be closed to continue the installation." + Environment.NewLine + "Do you want Setup to close SteamVR?",
+                "Close SteamVR?",
+                MessageBoxButton.YesNo,
+                MessageBoxImage.Warning
+                ) != MessageBoxResult.Yes)
+            {
+                //Only exit setup if SteamVR is still running after choosing "No"
```

Comment:
Can you add a MessageBox that says something like "The installer will now close" before that if it does close, because otherwise it looks like a crash.